### PR TITLE
Add blog post about building Docker 1.12 directly on a Pi

### PR DESCRIPTION
**Pull request by DieterReuter**
Signed-off-by: Dieter Reuter dieter.reuter@me.com

### CODE REVIEW

File: content/post/building-docker-directly-on-a-raspberry-pi.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+If we need a larger swap file, we could edit the config file
 `/etc/dphys-swapfile`,
+but for our use case the default 1.8 GByte swap file is sufficient.
+
+
+### Clone the Docker repo
+
+```
+$ cd
```
Comment (StefanScherer) [inline content/post/building-docker-directly-on-a-raspberry-pi.md]: The first `cd` can be skipped. Not relevant for the blog post.

File: content/post/building-docker-directly-on-a-raspberry-pi.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+$ git clone https://github.com/docker/docker.git
+$ cd ~/docker
```
Comment (StefanScherer) [inline content/post/building-docker-directly-on-a-raspberry-pi.md]: Use `cd docker`, works without the first `cd`.

Comment (DieterReuter) [inline content/post/building-docker-directly-on-a-raspberry-pi.md]: Thanks, will change it.

File: content/post/building-docker-directly-on-a-raspberry-pi.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+```
+$ git config --global user.email "dieter.reuter@me.com"

+$ git config --global user.name "Dieter Reuter"

+```
+
+So, let's cherry pick this specific pull request.
+```
+$ curl -sSL https://patch-diff.githubusercontent.com/raw/docker/docker/pull/25192.patch | git am
```
Comment (StefanScherer) [inline content/post/building-docker-directly-on-a-raspberry-pi.md]: My preferred way only with `git`is

```
git fetch origin pull/25192/head:fix-manpages-on-arm
git cherry-pick fix-manpages-on-arm
```

Comment (DieterReuter) [inline content/post/building-docker-directly-on-a-raspberry-pi.md]: OK, this seems to be a cleaner way. Will try it now, and then change the blog post.

Comment (StefanScherer) [inline content/post/building-docker-directly-on-a-raspberry-pi.md]: Learned from googling again and again and coming back to https://help.github.com/articles/checking-out-pull-requests-locally/

**Comment by StefanScherer** [comment]
LGTM 👍

**Pull request merged**